ctl/src/command.rs

```diff
@@ -385,3 +385,68 @@ pub fn status(channel: &mut Channel<ConfigMessage,ConfigMessageAnswer>) {
     }
   }
 }
+
+pub fn remove_backend(channel: &mut Channel<ConfigMessage,ConfigMessageAnswer>, app_id: &str, ip: &str, port: u16) {
+    let instance = Instance {
+      app_id: String::from(app_id),
+      ip_address: String::from(ip),
+      port: port
+    };
+
+    order_command(channel, Order::RemoveInstance(instance));
+}
+
+
+pub fn add_backend(channel: &mut Channel<ConfigMessage,ConfigMessageAnswer>, app_id: &str, ip: &str, port: u16) {
+  let instance = Instance {
+      app_id: String::from(app_id),
+      ip_address: String::from(ip),
+      port: port
+    };
+
+  order_command(channel, Order::AddInstance(instance));
+}
+
+fn order_command(channel: &mut Channel<ConfigMessage,ConfigMessageAnswer>, order: Order) {
```

Comment:
yes I will revert some change, but I think that we can factorise some code, but yes I will do a first unfactorised version